static/src/javascripts/projects/common/modules/experiments/tests/cmp-ccpa-test.js

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+
+export const ccpaCMPTest: ABTest = {
+    id: 'CmpCcpaOptIn',
```

Comment:
can we avoid the OptIn wording to avoid confusion? Maybe 'CmpCCPAInUs' ?

Reply:
This test is not restricted to US so I suggest avoid having that in the name.